tiled/adapters/tiff.py

```diff
@@ -59,6 +59,7 @@ def metadata(self):
         return d
 
     def read(self, slice=None):
+        raise Exception("BOOM!")
```

Comment:
Is this leftover from testing, or is it intentional?

Reply:
Yeah, this can't go into production. :)

Reply:
!